packages/genesys-spark-components/src/components/stable/gux-tooltip/tests/gux-tooltip.e2e.playwright.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  checkRenders,
+  expect,
+  test,
+  setContent
+} from '@test/playwrightTestUtils';
+import { renderConfig } from './gux-tooltip.common';
+
+test.describe('gux-tooltip', () => {
+  checkRenders({ renderConfigs: [renderConfig] });
+
+  test('Should render tooltip with correct attributes', async ({ page }) => {
+    await setContent(page, renderConfig.html);
+
+    const element = page.locator('#element');
+    const tooltip = page.locator('gux-tooltip');
+
+    let attrClass = await tooltip.getAttribute('class');
+    let tooltipClasses: string[] = attrClass ? attrClass.split(' ') : [];
+
+    expect(tooltipClasses).not.toContain('gux-show');
```

Comment:
I don't think you need to manually get the class list.
Try somethign like:
`await expect(tooltip).not.toContainClass('gux-show');`